docs/docs/reference/functions/middleware-and-helpers.md

```diff
@@ -51,7 +51,75 @@ export default async function corsHandler(req, res) {
 
 ## Custom body parsing
 
-Generally useful for file upload support.
+Generally useful for file upload support or raw bodies.
+
+Each function can export an object named `config`. This object will allow you to control the `body-parser` middleware used by Gatsby Functions. The `limit` property will allow configuration of payload size up to **_32mb_**.
+
+```js
+ // Config object
+ {
+   bodyParser: {
+     json: {},
+     raw: {},
+     text: {},
+     urlencoded: {}
+   }
+ }
+```
+
+Gatsby currently supports the `limit` and `type` options for the `bodyParser` configuration, which are documented by
+[body-parser](https://expressjs.com/en/resources/middleware/body-parser.html)
+
+### Examples
+
+```js
+// raw body (string)
+export const config = {
+  bodyParser: {
+    text: {
+      type: "*/*",
+    },
+  },
+}
+```
+
+```js
+// raw body (buffer)
+export const config = {
+  bodyParser: {
+    raw: {
+      type: "*/*",
+    },
+  },
+}
+```
+
+```js
+// limit payload to 10kb
+export const config = {
+  bodyParser: {
+    json: {
+      limit: "10kb",
+    },
+  },
+}
+```
 
-This is not yet supported. [Add a comment in the discussion if this is an
-important use case for you](https://github.com/gatsbyjs/gatsby/discussions/30735).
+```js
+// disable bodyParser
+export const config = {
+  bodyParser: false,
+  },
+}
+```
+
+```ts
+// TypeScript
+export const config: GatsbyFunctionConfig = { 
```

Comment:
You'll want to add the import for the type, otherwise people don't know where to get this type from